libraries/BluetoothSerial/examples/SerialToSerialBTM/SerialToSerialBTM.ino

```diff
@@ -1,52 +1,89 @@
-//This example code is in the Public Domain (or CC0 licensed, at your option.)
-//By Victor Tchistiak - 2019
+// This example code is in the Public Domain (or CC0 licensed, at your option.)
+// By Victor Tchistiak - 2019
 //
-//This example demostrates master mode bluetooth connection and pin 
-//it creates a bridge between Serial and Classical Bluetooth (SPP)
-//this is an extention of the SerialToSerialBT example by Evandro Copercini - 2018
+// This example demonstrates master mode Bluetooth connection to a slave BT device using PIN (password)
+// defined either by String "slaveName" by default "OBDII" or by MAC address
 //
+// This example creates a bridge between Serial and Classical Bluetooth (SPP)
+// This is an extension of the SerialToSerialBT example by Evandro Copercini - 2018
+//
+// DO NOT try to connect to phone or laptop - they are master
+// devices, same as the ESP using this code - it will NOT work!
+//
+// You can try to flash a second ESP32 with the example SerialToSerialBT - it should
+// automatically pair with ESP32 running this code
 
 #include "BluetoothSerial.h"
 
+#define USE_NAME // Comment this to use MAC address instead of a slaveName
+//#define USE_PIN // Uncomment this to use PIN during pairing. The pin is specified on the line below
+const char *pin = "1234"; // Change this to reflect the pin expected by the real slave BT device
+
+#ifndef USE_NAME
+  #define USE_MAC
+#endif
```

Comment:
Since you either have `USE_NAME` defined, or not, I suggest removing this and using `#ifdef`+`#else` in the code below. Else it looks like you could have both defined at the same time.